lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -804,8 +804,11 @@ bool getParameterTypes(Function *F, SmallVectorImpl<Type *> &ArgTys,
       HasSret = true;
       unsigned AS = Arg.getType()->getPointerAddressSpace();
       if (auto *STy = dyn_cast<StructType>(Ty))
-        ArgTys.push_back(
-            TypedPointerType::get(GetStructType(STy->getName()), AS));
+        if (!STy->isLiteral())
```

Comment:
Can we not just call STy->hasName() here? Suppose that will be more generic? Thanks